code-challenges.js

```diff
@@ -4,39 +4,84 @@
 
 // Pseudo coding is required.
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters. Use the two sets of test variables provided.
-
-// Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
-
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+// INPUT
+// create a funtion that takes a parameter
+// use conditional statement to set the degrees for each number
+// if number is 42, then it is below boiling point
+// if is it 212, the its is at boiling point
+// 350 is condidered above boiling point
+
+// OUTPUT
+// create a console log and use function that's been created
+// use the const temperature to use as parameters
+
+const tempNumber = (temperature) => {
+    if (temperature === 42) {
```

Comment:
Great attempt! I can see that you understand the concepts. Now let's dive into how to apply it dynamically. This question stated Boiling point is 212 degrees Fahrenheit. So consider conditionals, (temp === 212) for boiling point, (temp > 212) for above boiling point, (temp < 212) for below boiling point.